src/cs/core/Inventory.cs

```diff
@@ -0,0 +1,77 @@
+/**
+	Detective-style problem-solving magical rpg
+	Copyright (C) 2023 Rainy Day Studio
+
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+using Godot;
+using Microsoft.VisualBasic;
+using System;
+using System.Diagnostics;
+
+// Models the player's inventory
+public partial class Inventory : Node2D {
+
+	// ==================== Inventory Definitions=================
+	// Number of inventory slots
+	const int INVENTORY_SIZE = 3*7;
```

Comment:
You should export this to allow for it to be edited in Godot by a game designer. Ideally you would build out the sprite dynamically based on this value, but this is fine as placeholder for now